tests/queries/0_stateless/02968_file_log_multiple_read.sh

```diff
@@ -48,11 +48,21 @@ function count()
 	echo $COUNT
 }
 
-for i in {1..10}
-do
-	[[ $(count) -gt 0 ]] && break
-	sleep 1
-done
+function wait_for_row_count()
+{
+    local threshold="$1"
+    local timeout=15
```

Comment:
Let's increase it to 30

Whle the root cause is likely due to this test was waiting for some other S3 write to complete - https://github.com/ClickHouse/ClickHouse/pull/76137 - this should help.